oggm-edu/flowline_model.ipynb

```diff
@@ -88,8 +88,8 @@
     "# define horizontal resolution of the model:\n",
```

Comment:
<p>This should be:</p><p><br></p><p>```python</p><p>wider_widths = np.copy(widths)</p><p>```</p><p><br></p><p>otherwise `wider_widths` is a view and `wider_widths[0:15]` will change `widths` as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OGGM/oggm-edu-notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Right, yes.</p>